src/cocoa/toga_cocoa/widgets/base.py

```diff
@@ -14,6 +14,18 @@ def _set_container(self, container):
             self._constraints._container = container
         self.rehint()
 
+    def _set_hidden(self, status, children):
+        for view in self._container._impl.subviews:
+            [view.setHidden_(status) for c in children if c._impl == view]
```

Comment:
I'm also slightly confused why this is a list comprehension... using a list comprehension just for the side-effects seems odd to me.

While I'm here, where is `setHidden_` defined? I couldn't find it in a grep of the codebase?